dlrover/python/tests/test_elastic_training_agent.py

```diff
@@ -131,8 +131,8 @@ def setUp(self) -> None:
             rdzv_handler=self.rdzv_handler,
             max_restarts=self.config.max_restarts,
             monitor_interval=self.config.monitor_interval,
-            redirects=self.config.redirects,
-            tee=self.config.tee,
+            # redirects=self.config.redirects,
+            # tee=self.config.tee,
```

Comment:
It is better to remove those codes which are not needed.

Reply:
by mistake, reverted